checkstyle/suppressions.xml

```diff
@@ -85,6 +85,9 @@
     <suppress checks="NPathComplexity"
               files="SslTransportLayer.java"/>
 
+    <suppress checks="ImportControl"
+              files="KafkaFuture.java|KafkaFutureTest.java"/>
```

Comment:
This is necessary to implement `KafkaFuture#allOf` and `KafkaFuture#completedFuture`.